api/options/rest-api.php

```diff
@@ -28,23 +28,27 @@
 );
 
 register_woocommerce_admin_test_helper_rest_route(
-	'/options/(?P<option_id>\d+)',
+	'/options/(?P<option_names>(.*)+)',
 	'wca_test_helper_delete_option',
 	array(
 		'methods' => 'DELETE',
 		'args'                => array(
-			'option_id' => array(
-				'type'              => 'integer',
-				'sanitize_callback' => 'absint',
+			'option_names' => array(
+				'type'              => 'string',
 			),
 		),
 	)
 );
 
 function wca_test_helper_delete_option( $request ) {
 	global $wpdb;
-	$option_id = $request->get_param( 'option_id' );
-	$query     = $wpdb->prepare( "delete from {$wpdb->prefix}options where option_id = %d", $option_id );
+	$option_names = explode(',', $request->get_param( 'option_names' ));
```

Comment:
We don't have any linting, but we should probably stick with WP conventions for spacing around calls etc.

Reply:
Ignore this, I tweaked it myself (hope that's ok :-) )